src/js/api.js

```diff
@@ -1,80 +0,0 @@
-
-var development = true; 
-
-/******************************
-         api.js 
- ******************************/
-
-// Define global DV (Data Visualization) Configurations
-
-var DV = {
-	
-  // Define common CRUD functions
-  api : {},
-
-  // Is this a development configuration?
-  // Disables logging, and debug information
-  // for production environments.
-  development : development,
-
-  // Target API
-  url : (development) ? "http://localhost:8080/" : null,
-};
-
-/******************************
-      CRUD Functionality
- ******************************/
-
-// Reference on XHR callbacks
-// http://stackoverflow.com/questions/5485495/how-can-i-take-advantage-of-callback-functions-for-asynchronous-xmlhttprequest
-
-// GET
-// takes target URL, callback, callback
-DV.api.get = function(url, success, error){
```

Comment:
Do you imagine using our API to fetch tweet data? We'd use this functionality:

```javascript
DV.api.get("/twitter/search/{{hash}}", success(), error())
```
... Or would you prefer to save the pre-cached twitter datasets locally?